tools/CopySchema/Program.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Policy;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace CopySchema
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length != 2)
+            {
+                Console.Error.WriteLine("Expected 2 arguments, path to a manifest.json file, path to schema directory");
+                Environment.Exit(1);
+            }
+
+            string manifestFile = args[0];
+            string schemaDirectory = args[1];
+
+            if (!File.Exists(manifestFile))
```

Comment:
It might be nice to refactor the validation into its own function: `bool ValidateInput(...)`